src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+   "metadata": {},
+   "source": [
+    "Let us fix the split fraction of the purge stream from the splitter S101 and the outlet pressure from the compressor C101"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fix the split fraction of the 'purge' stream from S101\n",
+    "m.fs.S101.split_fraction[0, \"purge\"].fix(0.2)\n",
+    "\n",
+    "# Fix the pressure of the outlet from the compressor C101\n",
+    "m.fs.C101.outlet.pressure.fix(350000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, let us fix the temperature of the outlet from H102 and the pressure drop in H102 as the following"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fix the temperature of the outlet from the heater H102\n",
+    "m.fs.H102.outlet.temperature.fix(375)\n",
+    "\n",
+    "# Fix the pressure drop in the heater H102\n",
+    "m.fs.H102.deltaP.fix(-200000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "We have now defined all the feed conditions and the inputs required for the unit models. The system should now have 0 degrees of freedom i.e. should be a square problem. Please check that the degrees of freedom is 0. \n",
+    "\n",
+    "Use Shift+Enter to run the cell once you have typed in your code. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "exercise"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Check the degrees of freedom\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Check the degrees of freedom\n",
+    "print(degrees_of_freedom(m))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check the degrees of freedom\n",
+    "assert degrees_of_freedom(m) == 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialization\n",
+    "\n",
+    "This subsection will demonstrate how to use the built-in sequential decomposition tool to initialize our flowsheet.\n",
+    "\n",
+    "Let us first create an object for the `SequentialDecomposition` and specify our options for this. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seq = SequentialDecomposition()\n",
+    "seq.options.select_tear_method = \"heuristic\"\n",
+    "seq.options.tear_method = \"Wegstein\"\n",
+    "seq.options.iterLim = 5\n",
```

Comment:
Seeing as we don't need to fully converge the flowsheet, I would suggest decreasing this to 3.